templates/404.html

```diff
@@ -47,26 +49,34 @@
         <main id="maincontent">
             <div class="generic-intro">
                 <div class="container">
```

Comment:
Can/should this be a `tna-container` with a `tna-column tna-column--full` column in it? Is the layout as expected?

Reply:
It was providing the expected layout, but I've updated it to use the `tna-container` for consistency with the rest of the project. I'll make the same changes to the 500 template

For my own reference. Is the bootstrap container and grid a legacy tool that's being phased out?

Reply:
Yes it is 👍🏼